chart/templates/backends/ollama.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.backendRuntime.install -}}
+apiVersion: inference.llmaz.io/v1alpha1
+kind: BackendRuntime
+metadata:
+  labels:
+    app.kubernetes.io/name: backendruntime
+    app.kubernetes.io/part-of: llmaz
+    app.kubernetes.io/created-by: llmaz
+  name: ollama
+spec:
+  commands:
+    - sh
+    - -c
+  image: ollama/ollama
+  version: latest
+  # Do not edit the preset argument name unless you know what you're doing.
+  # Free to add more arguments with your requirements.
+  args:
+    - name: default
+      flags:
+        - "ollama serve &
```

Comment:
We need to support readiness and liveness next, see https://github.com/InftyAI/llmaz/issues/21, but  this is ok for now.

Reply:
let's support probe  in the future